docs/search/full-text/bm25.mdx

```diff
@@ -145,6 +139,22 @@ Use parentheses to group terms and control the order of operations.
 '(description:keyboard OR category:toy) AND description:metal'
 ```
 
+### Slop Operator
+
+The `~` slop operator is used to match phrases separated by words in between. For instance, let's say
+there exists a row with `description` set to "ergonomic metal keyboard." Because words "ergonomic" and "keyboard"
```

Comment:
Silly but should the "." be outside the " in "ergonomic metal keyboard." ?

Reply:
Yes